src/review.rs

```diff
@@ -35,8 +35,10 @@ pub struct Reviews {
     author: String,
 }
 
+#[derive(Debug)]
 pub enum Approval {
     Required,
+    #[allow(dead_code)]
```

Comment:
Could probably remove this enum altogether. I can look into that in a follow-up PR.